internal/util/testing/testing.go

```diff
@@ -18,12 +18,21 @@ import (
 	"net"
```

Comment:
unfortunately just about all of the packages under `util` use a standard package name already; it would be a pretty big undertaking to update them and all of their referencing imports. I'd rather do that all in a separate PR, and stick to the same script for now. If it makes you feel any better, the util subpackages are generally imported into the other packages using an alias, like `tt "github.com/Comcast/trickster/internal/util/testing"`, and where they are not, i usually clean it up.

Reply:
👍